### ISSUE TITLE
[IDEA] 

### ISSUE THREAD

**Issue by voltek62** [khoj-ai/khoj#884]
I would like to request a feature that allows users to send multiple notes directly into Obsidian. This feature would greatly enhance productivity and streamline the workflow for users who manage a large number of notes or frequently need to import batches of notes.

**Comment by debanjum** [comment]
Hey @voltek62, thanks for raising an issue for this. Can you clarify what you mean by import/send multiple notes directly into Obsidian? 

Do you want to send a subset of notes from your Obsidian vault to sync with your Khoj? Or do you want to limit the notes you want Khoj to reference when responding to your chat message? Or something else?

**Comment by voltek62** [comment]
Yes, that's correct. I would like to send a specific subset of notes from my Obsidian vault to sync with Khoj. This would allow me to manage which notes are referenced when Khoj responds to my queries, ensuring that only the most relevant information is used. This selective synchronization would help maintain a more organized and focused workflow, particularly when dealing with large vaults or when I only want to work with a particular set of notes.

**Comment by debanjum** [comment]
Hey @voltek62, thanks for clarifying the use-case! We support file filters in Khoj which maybe satisfactory for your requirements. To organize conversations with a specific set of notes, you can create conversations and limit them to access only the specified set of notes using the file filter dropdown on the left side panel of the web app. 

Would something like that within Obsidian satisfy your above use-case? 
If so, I'll focus this issue into improving file filter support from within the Khoj Obsidian plugin.

_P.S: We should also have another issue to limit files synced with Khoj from the Obsidian plugin. Other folks have requested the same so as to allow images to not be automatically synced with Khoj (and hit their data limits)_
